src/main/java/com/example/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package com.example.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
```

Comment:
its better, google the difference
```suggestion
    @NotBlank
    private String title;
    @NotBlank
    private String author;
    @NotBlank
```